test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -479,14 +348,19 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 		})
 
 		It("[test_id:49147] should run infra containers on reserved CPUs", func() {
-			var err error
+			var cpusetPath string
 			// find used because that crictl does not show infra containers, `runc list` shows them
 			// but you will need somehow to find infra containers ID's
 			podUID := strings.Replace(string(testpod.UID), "-", "_", -1)
-
 			podCgroup := ""
+			if cgroupV2 {
+				cpusetPath = "/rootfs/sys/fs/cgroup/kubepods.slice"
+			} else {
+				cpusetPath = "/rootfs/sys/fs/cgroup/cpuset"
```

Comment:
On v1 the `kubepods.slice` part exists as well

Reply:
correct, on v1 the paths differ